MSAL/src/instance/MSALAuthority.h

```diff
@@ -60,7 +60,7 @@ typedef void(^MSALAuthorityCompletion)(MSALAuthority *authority, NSError *error)
 
 @property MSALAuthorityType authorityType;
 @property NSURL *canonicalAuthority;
-@property BOOL validateAuthority;
+@property BOOL validatedAuthority;
```

Comment:
In objC style "isValidatedAuthority" would be the name of the accessor for a BOOL of validatedAuthority